pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -485,6 +485,7 @@ class Build {
     Run the Sign downstream job. We run this job on windows and jdk8 hotspot & jdk13 mac builds.
     The job code signs and notarizes the binaries so they can run on these operating systems without encountering issues.
     */
+    def SIGN_NODE_LABEL = DEFAULTS_JSON['signNodeLabel']
```

Comment:
It is better to move this definition to the function [build()](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/common/openjdk_build_pipeline.groovy#L1533) to be visible to both sign() and  signInstaller()